### PR TITLE
feat(src/components, src/pages): add schedule

### PR DESCRIPTION
**Pull request by Jumer-Caragay**

### CODE REVIEW

**Comment by Jumer-Caragay** [comment]
https://gyazo.com/cabe1c065fbe8603f8ac9c417bbe2d13

File: src/components/data/Schedule.json [hunk 1]
```diff
@@ -5,59 +5,208 @@
       "events": 
       [
         {
-          "title": "Paper Airplane Activity",
-          "location": "TBD"
+          "title": "Sponsor Check-in",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Can you remove this block? This is not needed for attendees

File: src/components/data/Schedule.json [hunk 2]
```diff
@@ -5,59 +5,208 @@
...
+          "time": "4:00 PM",
+          "location": "Front Desk",
+          "id": 1
         },
         {
-          "title": "Breakfast",
-          "location": "TBD"
+          "title": "Dinner",
+          "time": "5:30 PM",
+          "location": "Help Desk",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Front Desk

File: src/components/data/Schedule.json [hunk 3]
```diff
@@ -5,59 +5,208 @@
...
+          "id": 2
+        },
+        {
+          "title": "Intro to iOS",
+          "time": "6:00 PM",
+          "location": "UU220",
```
Comment (selynna) [inline src/components/data/Schedule.json]: UU 220

File: src/components/data/Schedule.json [hunk 4]
```diff
@@ -5,59 +5,208 @@
...
+          "time": "9:00 PM",
+          "location": "Everywhere",
+          "id": 5
         },
         {
-          "title": "Lunchn",
-          "location": "TBD"
+          "title": "Team Mixer",
+          "time": "9:00 PM",
+          "location": "UU220",
```
Comment (selynna) [inline src/components/data/Schedule.json]: UU 220

File: src/components/data/Schedule.json [hunk 5]
```diff
@@ -5,59 +5,208 @@
...
         }
       ]
     },
     {
       "events":
       [
         {
-          "title": "Cup Stacking Challenge",
-          "location": "TBD"
+          "title": "Late Night Snacks",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Late Night Snack

File: src/components/data/Schedule.json [hunk 6]
```diff
@@ -5,59 +5,208 @@
...
         },
         {
           "title": "Breakfast",
-          "location": "TBD"
+          "time": "8:00 AM",
+          "location": "Front Desk",
+          "id": 10
+        },
+        {
+          "title": "Creating The Perfect Pitch On Short Notice",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Isn't this Sunday morning?

File: src/components/data/Schedule.json [hunk 7]
```diff
@@ -5,59 +5,208 @@
...
+          "id": 11
+        },
+        {
+          "title": "F5 Networks Workshop",
+          "time": "10:00 AM",
+          "location": "UU220",
+          "id": 12
+        },
+        {
+          "title": "Scovil Workshop",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Machine Learning in iOS

File: src/components/data/Schedule.json [hunk 8]
```diff
@@ -5,59 +5,208 @@
...
+          "id": 24
+        },
+        {
+          "title": "Project Submissions Due",
+          "time": "8:00 AM",
+          "location": "Everywhere",
+          "id": 25
+        },
+        {
+          "title": "Judges' Meeting",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Remove this block

File: src/components/data/Schedule.json [hunk 9]
```diff
@@ -5,59 +5,208 @@
...
+          "id": 27
+        },
+        {
+          "title": "Judging",
+          "time": "9:30 AM",
+          "location": "Chumash Auditorium",
+          "id": 28
+        },
+        {
+          "title": "Judging Ends",
```
Comment (selynna) [inline src/components/data/Schedule.json]: Remove this block

**Review by selynna — CHANGES_REQUESTED** [review]

File: src/components/data/Schedule.json (hunk 6 above)
Comment (selynna) [reply src/components/data/Schedule.json]: @Jumer-Caragay  just double checked my email, this is 9am Sunday. Sorry :(

**Review by selynna — CHANGES_REQUESTED** [review]
Cool, just one update I need from you! Otherwise it's good to go, but I do have a question I'll bring up on Slack.

**Pull request merged**